src/lib/services/youtube/api.ts

```
@@ -331,13 +323,7 @@ export async function createPlaylistWithTracks(
       playlistId,
     };
   } catch (error) {
-    console.error("[PLAYLIST] Error creating playlist:", error);
-    return {
-      added: 0,
-      failed: tracks.length,
-      total: tracks.length,
-      playlistId: null,
-    };
+    throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)